AsyncDisplayKit/Details/ASIndexedNodeContext.h

```diff
@@ -0,0 +1,22 @@
+//
+//  ASIndexedNodeContext.h
+//  AsyncDisplayKit
+//
+//  Created by Huy Nguyen on 2/28/16.
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+
+#import <AsyncDisplayKit/ASDataController.h>
+#import <AsyncDisplayKit/ASDimension.h>
+
+@interface ASIndexedNodeContext : NSObject
+
+@property (nonatomic, readonly, strong) ASCellNodeBlock nodeBlock;
+@property (nonatomic, readonly, assign) NSIndexPath *indexPath;
```

Comment:
Fix here: https://github.com/facebook/AsyncDisplayKit/pull/1290. Will keep updating that PR to address coming feedbacks.